cloud/services/object_storage_buckets.go

```diff
@@ -18,9 +17,11 @@ func EnsureObjectStorageBucket(ctx context.Context, bScope *scope.ObjectStorageB
 		bScope.Bucket.Spec.Cluster,
 		bScope.Bucket.Name,
 	)
-	linodeErr := &linodego.Error{}
-	if errors.As(err, linodeErr) && linodeErr.StatusCode() != http.StatusNotFound {
-		return nil, fmt.Errorf("failed to get bucket from cluster %s: %w", bScope.Bucket.Spec.Cluster, err)
+	if err != nil {
+		linodeErr := linodego.NewError(err)
+		if linodeErr.StatusCode() != http.StatusNotFound {
```

Comment:
I think for this we should be able to use `util.IgnoreLinodeAPIError(err, http.StatusNotFound)`

Reply:
Use I was able to use that. Thanks for pointing that out!